### PR TITLE
Validation shift 2 and removed TOTP Setup

### PR DESCRIPTION
**Pull request by therealssj**
Removed the totp setup and moved it to ga login

### CODE REVIEW

File: src/Form/BasicDisable.php [hunk 1]
```diff
@@ -15,11 +16,36 @@
...
+  protected $manager;
+
+  /**
+   * Provides the user data service object.
+   *
+   * @var \Drupal\user\UserDataInterface
+   */
+  protected $userData;
+
+  public static function create(ContainerInterface $container) {
```
Comment (nerdstein) [inline src/Form/BasicDisable.php:38]: Missing functional comments

File: src/Form/BasicSetup.php [hunk 2]
```diff
@@ -12,30 +12,43 @@
...
    */
   protected $manager;
 
+  /**
+   * Provides the user data service object.
+   *
+   * @var \Drupal\user\UserDataInterface
+   */
+  protected $userData;
+
```
Comment (nerdstein) [inline src/Form/BasicSetup.php:41]: Missing functional comments

File: tfa_basic.services.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+services:
```
Comment (nerdstein) [inline tfa_basic.services.yml:1]: can we eliminate this file?

File: src/Form/BasicDisable.php (hunk 1 above)
Comment (therealssj) [inline src/Form/BasicDisable.php:38]: Aah yes I have done all that in the next PR.

**Pull request merged**